docs/core/diagnostics/diagnostics-in-containers.md

```diff
@@ -0,0 +1,69 @@
+---
+title: Collect diagnostics in containers
+description: In this article, you'll learn how .NET Core diagnostics tools can be used in Docker containers.
+ms.date: 09/01/2020
+---
+
+# Collect diagnostics in containers
+
+**This article applies to: ✔️** .NET Core 2.1 SDK and later versions
+
+The same diagnostics tools that are useful for diagnosing .NET Core issues in other scenarios also work in Docker containers. However, some of the tools require special steps to work in a container. This article covers how tools for gathering performance traces and collecting dumps can be used in Docker containers.
+
+## Using .NET Core global CLI tools in a container
```

Comment:
This header is very large, it will wrap on the published page.

Reply:
I've updated it to `## Using .NET Core CLI tools in a container` which is a bit shorter. Let me know if you think I need to trim more.